### DISCUSSION TITLE
The best practice to organize the unused extensions?

### DISCUSSION THREAD

**Discussion by nothingness6** [microsoft/vscode-discussions#601] [Extension Development QnA]
I'm using the portable version and found the files in the data folder are quite big. I'd like to organize (delete) them AUTOMATICALLY in VScode. 
Is there a way for that?

VS Code version: Code 1.77.3
OS version: Windows 10 64bit 22H2
![vscode](https://github.com/microsoft/vscode-discussions/assets/105210258/d0ef3136-66d4-4cc9-9703-6a29d08ddb37)

**Comment by isidorn** [comment]
This discussion is for extension authors. This feels like a general VS Code question. I suggest you reach out to stack overflow for those.
